Fix bug with phase1-up.sh script in roadwarrior case 

--- ipsec-tools-0.7.3.orig/src/racoon/isakmp.c	2010-04-12 16:45:38.000000000 +0200
+++ ipsec-tools-0.7.3/src/racoon/isakmp.c	2010-04-12 16:47:19.000000000 +0200
@@ -1679,6 +1679,20 @@
 		}
 #endif
 
+		if (setsockopt(p->sock, SOL_SOCKET,
+#ifdef __linux__
+					 SO_REUSEADDR,
+#else
+					 SO_REUSEPORT,
+#endif
+					 (void *)&yes, sizeof(yes)) < 0) {
+			plog(LLV_ERROR, LOCATION, NULL,
+				"failed to set REUSE flag on %s (%s).\n",
+				saddr2str(p->addr), strerror(errno));
+			close(p->sock);
+			goto err_and_next;
+		}
+
 		if (setsockopt_bypass(p->sock, p->addr->sa_family) < 0)
 			goto err_and_next;
 
